lib/sql.js

```diff
@@ -229,7 +230,7 @@ SQLConnector.prototype.tableEscaped = function(model) {
  * @returns {String} The escaped column name
  */
 SQLConnector.prototype.columnEscaped = function(model, property) {
-  return this.escapeName(this.column(model, property));
+  return this.tableEscaped(model) + '.' + this.escapeName(this.column(model, property));
 };
```

Comment:
This change will unfortunately lead to wrong DDL code for autoupdate/automigrate. As it'll additionally qualify column names with table name (+ schema name for PostgreSQL).

Example:

```
CREATE TABLE FOO {
  `FOO`.`ID` INT,
  `FOO`.`BAR` VARCHAR...
}
```

see this ``` `FOO`. ``` thing?